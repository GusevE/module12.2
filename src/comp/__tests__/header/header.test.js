import React from "react";
import {render} from '@testing-library/react'
import Header from "../../../component/header/header"

it ('Тест записи', ()=> {
    const component = render(<Header />);
    expect (component).toMatchSnapshot();

});
