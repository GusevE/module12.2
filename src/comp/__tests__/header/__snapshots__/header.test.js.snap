// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Тест записи 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="container"
      >
        <div
          class="header"
        >
          <div
            class="header__logo"
          >
             netflixroulette
          </div>
          <div
            class="header__title"
          >
             Find your movie
          </div>
          <label>
            <input
              class="header__input"
              placeholder="Quentin Terantino"
              type="search"
            />
          </label>
          <div
            class="search"
          >
            <div
              class="search__title"
            >
              search by 
            </div>
            <button
              class="btn active"
            >
               Title
            </button>
            <button
              class="btn"
            >
               Genre
            </button>
            <button
              class="btn active"
              id="btn__search"
            >
               search
            </button>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="container"
    >
      <div
        class="header"
      >
        <div
          class="header__logo"
        >
           netflixroulette
        </div>
        <div
          class="header__title"
        >
           Find your movie
        </div>
        <label>
          <input
            class="header__input"
            placeholder="Quentin Terantino"
            type="search"
          />
        </label>
        <div
          class="search"
        >
          <div
            class="search__title"
          >
            search by 
          </div>
          <button
            class="btn active"
          >
             Title
          </button>
          <button
            class="btn"
          >
             Genre
          </button>
          <button
            class="btn active"
            id="btn__search"
          >
             search
          </button>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
